contents/blog/sales-no-experience.md

```diff
@@ -0,0 +1,142 @@
+---
+date: 2024-09-30
+title: How to do sales with no experience
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author:
+  - james-hawkins
+---
+
+"I just need to hire a sales person so we get some revenue". There it is people, the start of the end.
+
+Here's the rest of the movie:
+
+* salesperson starts
+* they get a few meetings, they're pretty random feeling
+* you start building stuff that people in the meetings have talked about
+* nothing happens for ages, or at all
+* you now don't know if the salesperson is at fault, or if you don't have product market fit
```

Comment:
This paragraph later:

> Everything else is a distraction early on. So what are you doing if you don't care about this? Likely being avoidant from all the rejection and fear of failure! You may not want to hear it but a magical salesperson will be _far_ less likely to figure this out than you.

Especially the "magical salesperson" bit feels like a clearer hook. Something like telling the "The myth of the magical sales person" of that makes sense.